app/models/ticket.rb

```diff
@@ -31,6 +31,10 @@ def identifier
     "%s:%06x" % [project.shortname, sequential_id]
   end
 
+  def identifier=(identifier)
+    super(identifier)
```

Comment:
What does that do?

Reply:
From what I understand it is an explicit setter.

I first had it without the `=` and that failed with stack overflow or something like it.